### PR TITLE
[Merged by Bors] - feat: some lemmas about nnnorm and integration / L1-norm

### PR DESCRIPTION
**Pull request by fpvandoorn**
* From the Sobolev project

Co-authored-by: Heather Macbeth 25316162+hrmacbeth@users.noreply.github.com

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Integral/Bochner.lean [hunk 1]
```diff
@@ -734,6 +734,9 @@ theorem norm_Integral_le_one : ‖integralCLM (α := α) (E := E) (μ := μ)‖
   norm_setToL1_le (dominatedFinMeasAdditive_weightedSMul μ) zero_le_one
 #align measure_theory.L1.norm_Integral_le_one MeasureTheory.L1.norm_Integral_le_one
 
+theorem nnnorm_Integral_le_one : ‖integralCLM (α := α) (E := E) (μ := μ)‖₊ ≤ (1 : ℝ) :=
```
Comment (eric-wieser) [inline Mathlib/MeasureTheory/Integral/Bochner.lean:737]:
```suggestion
theorem nnnorm_Integral_le_one : ‖integralCLM (α := α) (E := E) (μ := μ)‖₊ ≤ (1 : ℝ≥0) :=
```

File: Mathlib/MeasureTheory/Integral/Bochner.lean [hunk 2]
```diff
@@ -971,6 +977,14 @@ theorem norm_integral_le_lintegral_norm (f : α → G) :
...
+theorem nnnorm_integral_le_lintegral_nnnorm (f : α → G) : ‖∫ x, f x ∂μ‖₊ ≤ ∫⁻ x, ‖f x‖₊ ∂ μ := by
+  rw [integral_def]
+  split_ifs with hG hf
+  · calc _ ≤ (‖(Integrable.toL1 f hf)‖₊ : ℝ≥0∞) := by norm_cast; apply L1.nnnorm_integral_le
+      _ = _ := hf.nnnorm_toL1
+  · simp
+  · simp
+
 theorem ennnorm_integral_le_lintegral_ennnorm (f : α → G) :
     (‖∫ a, f a ∂μ‖₊ : ℝ≥0∞) ≤ ∫⁻ a, ‖f a‖₊ ∂μ := by
```
Comment (eric-wieser) [inline Mathlib/MeasureTheory/Integral/Bochner.lean:989]: Can this be golfed using the new result?

**Review by eric-wieser — APPROVED** [review]
bors d+

I've not attempted to golf the proofs, but modulo one small error, the statements all look great.

File: Mathlib/MeasureTheory/Integral/Bochner.lean (hunk 1 above)
Comment (eric-wieser) [reply Mathlib/MeasureTheory/Integral/Bochner.lean:737]: Or just
```suggestion
theorem nnnorm_Integral_le_one : ‖integralCLM (α := α) (E := E) (μ := μ)‖₊ ≤ 1 :=
```

File: Mathlib/MeasureTheory/Integral/Bochner.lean (hunk 2 above)
Comment (fpvandoorn) [reply Mathlib/MeasureTheory/Integral/Bochner.lean:989]: Oops! It's the same result.

**Pull request closed without merging**